src/firejail/main.c

```diff
@@ -982,6 +982,14 @@ int main(int argc, char **argv, char **envp) {
 	int arg_caps_cmdline = 0; 	// caps requested on command line (used to break out of --chroot)
 	char **ptr;
 
+#ifndef HAVE_SUID
+	if (geteuid() != 0) {
+		fprintf(stderr, "Error: Firejail need to be SUID.");
```

Comment:
nitpick/typo: Error: Firejail needs to be SUID.

Reply:
fixed, thanks.